robots/pkg/flakefinder/report_data.go

```diff
@@ -135,6 +135,8 @@ func CreateFlakeReportData(results []*JobResult, prNumbers []int, endOfReport ti
 		}
 	}
 
+	sort.Strings(headers)
```

Comment:
If I understand that correctly, this line ensures that we always generate the table columns in the same order?